query-engine/driver-adapters/src/result.rs

```diff
@@ -19,11 +19,11 @@ pub struct PostgresErrorDef {
 /// See driver-adapters/js/adapter-utils/src/types.ts file for example
 pub(crate) enum DriverAdapterError {
     /// Unexpected JS exception
-    GenericJsError {
+    GenericJs {
```

Comment:
As soon as we add 1 more variant here, Clippy starts complaining that all variants share same postfix, which is also part of enum name itself and so it is redundant. I kind of agree with sentient paperclip, and it's not quite public API yet, so I think it's ok to rename the variants. If we really want to lean on a safer side, we could either silence clippy here or ask serde to rename the variants when reading them from JS

Reply:
As long as the feature is preview, we can totally adapt this in any way we want. Now is the time to do so.